client/server/pages/index.js

```diff
@@ -830,12 +830,16 @@ function wpcomPages( app ) {
 	} );
 
 	app.get( [ '/subscriptions', '/subscriptions/*' ], function ( req, res, next ) {
-		if ( req.context.isLoggedIn ) {
+		const showSitesViewToInternalUsers = config.isEnabled(
+			'subscription-management/internal-users/sites-view'
+		);
+
+		if ( req.context.isLoggedIn && ! showSitesViewToInternalUsers ) {
```

Comment:
Just thinking about it again. 🤔 I am really confused why we do not simply remove this check. Without the need for a feature flag here at all. I think it would make sense to do so, right?

Reply:
But mind the feature flag is still needed for the tabs navigation.